typescript_utilities/src/wasmFunctions.ts

```diff
@@ -70,6 +70,7 @@ export interface WasmWorker {
     statusCode: number,
     payloadHeaders: HeaderFields,
     payloadBody: Uint8Array,
+    process_link: boolean,
```

Comment:
This function already has 10 aguments, and adding `process_link: bool` makes it to 11. I would suggest to add `process_link: bool` as a field of the [ConfigInput struct](https://github.com/google/sxg-rs/blob/42a537d39c1203df28ae9be099203a7927915fe0/sxg_rs/src/config.rs#L23), and use `self.config.process_link` at `sxg_rs/src/lib.rs`.

Reply:
One issue with putting the flag inside ConfigInput is that it needs to be mutable. For a site with mixed SXG and AMP SXG, then we need create different SxgWorker / ConfigInput. 

Perhaps we can make that argument optional (default to true). We can also refactor that function to take in a struct like in the native Rust code instead since 10 is also a lot.

Reply:
Awesome work. Thank you very much.